### DISCUSSION TITLE
Loading wavesurfer...

### DISCUSSION THREAD

**Discussion by DTBob** [katspaugh/wavesurfer.js#2863] [Q&A] [answered]
I've been loading wavesurfer like this, which was working....

	<script src="https://unpkg.com/wavesurfer.js"></script>
	<script src="https://unpkg.com/wavesurfer.js/dist/plugin/wavesurfer.microphone.min.js" type="text/javascript"></script>
	<script src="https://unpkg.com/wavesurfer.js/dist/plugin/wavesurfer.playhead.min.js" type="text/javascript"></script>
	<script src="https://unpkg.com/wavesurfer.js/dist/plugin/wavesurfer.cursor.min.js" type="text/javascript"></script>
	<script src="https://unpkg.com/wavesurfer.js/dist/plugin/wavesurfer.timeline.min.js" type="text/javascript"></script>

but the plugin links are no longer working. What is the correct url for plugins now?

Thanks!

**Comment by DTBob** [comment, marked as answer]
Sorry, I did not realise there was a new version. I updated the links to...
<script src="https://unpkg.com/wavesurfer.js@6.6"></script>
<script src="https://unpkg.com/wavesurfer.js@6.6/dist/plugin/wavesurfer.microphone.min.js" type="text/javascript"></script>
<script src="https://unpkg.com/wavesurfer.js@6.6/dist/plugin/wavesurfer.playhead.min.js" type="text/javascript"></script>
<script src="https://unpkg.com/wavesurfer.js@6.6/dist/plugin/wavesurfer.cursor.min.js" type="text/javascript"></script>
<script src="https://unpkg.com/wavesurfer.js@6.6/dist/plugin/wavesurfer.timeline.min.js" type="text/javascript"></script>

**Reply by katspaugh** [reply]
Hmm, I thought the old link you posted initially also worked. Thanks for pointing this out and finding a solution!
